docs/howto/eos_designs/network_services.md

```diff
@@ -0,0 +1,148 @@
+
+<!--
+  ~ Copyright (c) 2025 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Network Services
+
+## Introduction to Network Services
+
+**"Network Services"** is an abstracted data model that allows you to define and deploy Layer 2 (VLANs) and Layer 3 (VRF) services across your entire fabric.
+
+The model is designed to be tenant-based, meaning you group network services under a **"tenant"** which can represent an organization, department, or any logical container. This structure allows for the reuse of VLAN and SVI IDs across different tenants while ensuring that configurations are deployed only to the relevant devices in the fabric. While this allows for complex multi-tenant environments, it's important to note that for a simple setup, it's perfectly fine to have one single `tenant` that represents all of your services.
+
+## Core Concept: Tenants
+
+Everything starts with a tenant. A tenant is the top-level key under which you define all associated VRFs and VLANs.
+
+- **Grouping:** Tenants group VRFs and L2 VLANs.
+
+- **VNI Range:** For EVPN VXLAN fabrics, each tenant is assigned a `mac_vrf_vni_base`. This number is used to automatically calculate the VXLAN Network Identifier (VNI) for each VLAN/SVI within that tenant (e.g., `vni = mac_vrf_vni_base + vlan_id`).
+
+- **Filtering:** AVD uses a filtering model to control which devices a tenant's configuration is applied to. This is typically done using [tags](tags.md).
```

Comment:
I would rename to Network Services tags, so we don't confuse them with CloudVision tags. (If merging this suggestion `tags.md` needs to be renamed to `network_services_tags.md`)

```suggestion
- **Filtering:** AVD uses a filtering model to control which devices a tenant's configuration is applied to. This is typically done using [Network Services tags](network_services_tags.md).
```